src/vos/tests/evt_ctl.c

```diff
@@ -2395,17 +2293,46 @@ ts_cmd_run(char opc, char *args)
 		rc = 0;
 		break;
 	}
-	if (rc != 0)
-		D_PRINT("opc=%d failed with rc="DF_RC"\n", opc, DP_RC(rc));
-
+	if(st != NULL) {
+		rc = -1;
+	}
 	return rc;
 }
 
+static void
+ts_group(void ** state){
+	int	opc = 0;
+	while((opc = getopt_long(test_group_stop-test_group_start+1,
```

Comment:
(style)  Missing a blank line after declarations
(style)  space required before the open parenthesis '('